jumeg/connectivity/con_viz.py

```diff
@@ -290,7 +297,18 @@ def plot_connectivity_circle(con, node_names, indices=None, n_lines=None,
 
     # Draw lines between connected nodes, only draw the strongest connections
     if n_lines is not None and len(con) > n_lines:
-        con_thresh = np.sort(np.abs(con).ravel())[-n_lines]
+
+        n_significant_cons = len(np.where(con)[0])
+
+        if n_significant_cons > n_lines:
+            con_thresh = np.sort(np.abs(con).ravel())[-n_lines]
+        elif n_significant_cons == 0:
+            # there are no significant connections, set minimum threshold to
+            # avoid plotting everything
+            con_thresh = 0.001
+        else:
+            con_thresh = np.sort(np.abs(con).ravel())[-n_significant_cons]
```

Comment:
Suggestion to change n_significant_cons to n_present_cons or something else. significant makes one think of stats being applied somewhere and makes it confusing.